chapters/object_types/objects_in_world.txt

```diff
@@ -179,8 +179,8 @@ there are no surfaces, no volumes, or no lights instanced.
 [[instance]]
 === Instance
 
-Instances apply transforms to groups for placement in the World. Instances are
-created with
+Instances apply transforms to groups for placement in the World. All matrix transforms 
+are expected to use a row-major layout. Instances are created with
```

Comment:
Our discussion was  a bit inconclusive. In other places ANARI is compatible with OpenGL and glTF (using a right-handed coord system, screen space origin is lower left, ...), thus we should rather consider to move to column-major format (as actually defined in Sec 3.5 for the matrix types)

Reply:
Sounds good to me! I'll hold off until after our next WG meeting and hopefully have a more conclusive decision.